hnn_core/gui/_viz_manager.py

```diff
@@ -836,12 +837,6 @@ def data(self):
 
     def reset_fig_config_tabs(self, template_name=None):
         """Reset the figure config tabs with most recent simulation data."""
-        simulation_names = tuple(self.data['simulations'].keys())
-        for tab in self.axes_config_tabs.children:
-            controls = tab.children[1]
-            for ax_control in controls.children:
-                simulation_selection = ax_control.children[0]
-                simulation_selection.options = simulation_names
         # recover the default layout
```

Comment:
What is the role of this part of the function? Like when is it triggered in the GUI?

Reply:
It is executed after running a simulation or uploading data. It updates the first dropdown list in the second tab of a figure with simulation names. According to the logs in #543, the main idea was to make newly run simulations accessible to previously created figures. I think the PR was rushed and merged with the main branch without further discussion on whether this was the correct behavior. Looking at the screenshots in the PR, the GUI looks a bit different from the current layout.